README.md

```diff
@@ -29,6 +29,14 @@ joining our [public Slack channel.](https://adhocteam-public.herokuapp.com)
 We think a sign of a good developer is one who asks questions sooner, rather
 than later. We are happy to help guide you to the right solution.
 
+On Overengineering
+------------------
+
+Your goal on these exercises should be to write well-structured code that addresses the problems as they
+are stated in the instructions. Designing the code in an easy-to-read and straightforward way will get you
+more points than creating a fancy object hierarchy, showing off what you would do in a larger project,
+or making it enterprise ready (please don't make it enterprise ready).
```

Comment:
I laughed at "(please don't make it enterprise ready)", but worry that people could be confused or put off by that phrase